solr/modules/gcs-repository/build.gradle

```diff
@@ -51,6 +56,7 @@ dependencies {
     implementation('io.opencensus:opencensus-api') { transitive = false }
     implementation('io.opencensus:opencensus-contrib-http-util') { exclude group: 'com.google.guava', module: 'guava' }
 
+    testImplementation('com.google.code.findbugs:jsr305') { transitive = false }
```

Comment:
I tried removing this and everything still seemed to work.

Reply:
I moved this from "implementation" to "testImplementation". But if it is not needed, then fine.